### DISCUSSION TITLE
Allow managing storage adapter on a per-file basis

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#3076] [Feature Requests]
* Do we want folders to be scoped to a storage adapter?
* Do we show a dropdown every time you upload a folder?
* What about changing storage adapters on the fly?
* Can a single file be stored in multiple adapters at the same time?

**Comment by benhaynes** [comment]
* Do we want folders to be scoped to a storage adapter?
    * Nope. Just files.
* Do we show a dropdown every time you upload a folder?
    * No, should have a default and the ability to manually override
* What about changing storage adapters on the fly?
    * Not MVP, but would be nice to have eventually.
* Can a single file be stored in multiple adapters at the same time?
    * No, 1:1

**Reply by rijkvanzanten** [reply]
> Do we show a dropdown every time you upload a folder?
> * No, should have a default and the ability to manually override

So then we do show a dropdown, but it's pre-filled?

**Reply by benhaynes** [reply]
Kinda. If you use any of the seamless ways (drag and drop, etc) then it goes to the default. But if you open the upload modal I think there would be a minimal badge that says which storage adapter is the default... and clicking that would allow you to change it for that batch (yes, like a dropdown).

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
